BUILDING_ON_WINDOWS.md

```diff
@@ -20,6 +20,8 @@ Notes:
 1. Visit the [Qt Open Source Page](https://www.qt.io/download-open-source).
 2. Scroll down to the bottom
 3. Then select "Download the Qt Online Installer"
+4. Within the installer, Qt will prompt you to create an account (to access Qt downloads).
+5. When you get to the Installation Folder section within the Qt installer, you can create an empty folder named 'Qt' within your 'C:' directory and install Qt within that folder. (You cannot install Qt in the 'Program Files' folder because of the blank space (' ') between 'Program' and 'Files' and you cannot install Qt directly in 'C:'.)
```

Comment:
> and you cannot install Qt directly in 'C:

![image](https://github.com/Chatterino/chatterino2/assets/41973452/7d64bb9d-91b6-4b5f-a0b1-c63e374b7655)

Reply:
I have a 'Qt' folder in 'C:' as well but I believe I had to create it manually. It wouldn't let me pick 'C:' as a location, so I picked the newly created 'Qt' folder I made as the location.

Reply:
Ah I think I know what happened. I wrote down the error message when it happened and it said something about how the install directory is completely deleted. 

I went to another computer to download Qt and it turns out the directory is originally defaulted to 'C:\Qt' but when I pressed the 'Program Files' folder it changed to 'C:\Program Files' and when I pressed back on 'C:' that Qt folder was gone.

Reply:
> > and you cannot install Qt directly in 'C:
> 
> ![image](https://user-images.githubusercontent.com/41973452/271879712-7d64bb9d-91b6-4b5f-a0b1-c63e374b7655.png)

Thanks for spotting that.